### PR TITLE
Support JsDiagGetProperties on functionHandle returned in JsDiagGetStackTrace

### PR DESCRIPTION
**Pull request by agarwal-sandeep**
Support JsDiagGetProperties on functionHandle returned in JsDiagGetStackTrace

### CODE REVIEW

**Comment by agarwal-sandeep** [comment]
Fixes #3678 
@fatcerberus - FYI

**Comment by fatcerberus** [comment]
@agarwal-sandeep Thanks, this was quick :)

File: bin/ch/DbgController.js [hunk 1]
```diff
@@ -781,6 +781,44 @@ var controllerObj = (function () {
                     'sources': sources
                 });
             },
+            dumpFunctionProperties: function (frameIdOrArrayOfIds, expandLevel) {
```
Comment (akroshg) [inline bin/ch/DbgController.js]: >frameIdOrArrayOfIds, expandLevel [](start = 46, length = 32)

nit : you can use default to get rid of undefined checks below.

File: lib/Jsrt/JsrtDebuggerObject.cpp [hunk 2]
```diff
@@ -693,16 +710,47 @@ Js::DynamicObject * JsrtDebuggerObjectFunction::GetJSONObject(Js::ScriptContext
...
     JsrtDebugUtils::AddPropertyToObject(functionObject, JsrtDebugPropertyId::handle, this->GetHandle(), scriptContext);
 
     return functionObject;
 }
 
+Js::DynamicObject * JsrtDebuggerObjectFunction::GetChildren(Js::ScriptContext * scriptContext, uint fromCount, uint totalCount)
+{
+    if (this->objectDisplay == nullptr)
+    {
+        Js::ResolvedObject functionResolvedObject;
```
Comment (akroshg) [inline lib/Jsrt/JsrtDebuggerObject.cpp]: nit : initialize?

Comment (agarwal-sandeep) [reply lib/Jsrt/JsrtDebuggerObject.cpp]: It's not a pointer and have default initializer which gets called.

File: bin/ch/DbgController.js (hunk 1 above)
Comment (agarwal-sandeep) [reply bin/ch/DbgController.js]: Done

**Pull request merged**